DigitalOffice/Services/AuthenticationService/Broker/Requests/UserCredentialsInternalRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace LT.DigitalOffice.Broker.Requests
+{
+    public class UserCredentialsInternalRequest
+    {
+        public string Email { get; }
+        public Guid UserId { get; }
+        public string PasswordHash { get; }
+    }
```

Comment:
We deal with messages as interfaces, not classes. Correct all RabbitMQ requests/responses

Reply:
And I suggest leave the name as it is -- not to add "I".

For MassTransit, let's make an exception. I personally will be confused when writing that I get IResponse, so lets discuss

Reply:
Fixed